### PR TITLE
Migrate from android.support to androidx

### PR DESCRIPTION
**Pull request by voghDev**
This is a replication of [this pull request to the original author](https://github.com/gcacace/android-signaturepad/pull/141).
Tested it and it works fine in all my devices 👍

Feel free to post anything you'd like to have added to this PR

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,10 +1,11 @@
 buildscript {
     repositories {
+        google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.5.1'
```
Comment (jrgonzalezg) [inline build.gradle]: Latest stable is 3.5.3

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```
Comment (jrgonzalezg) [inline gradle/wrapper/gradle-wrapper.properties]: Is it possible to use latest Gradle 6.2?

File: gradle.properties [hunk 3]
```diff
@@ -33,6 +33,8 @@ POM_LICENCE_DIST=repo
...
 POM_DEVELOPER_NAME=Gianluca Cacace
 
-ANDROID_BUILD_TARGET_SDK_VERSION=25
-ANDROID_BUILD_TOOLS_VERSION=25.0.3
-ANDROID_BUILD_SDK_VERSION=25
+ANDROID_BUILD_TARGET_SDK_VERSION=29
+ANDROID_BUILD_SDK_VERSION=29
+
+android.useAndroidX=true
+android.enableJetifier=true
```
Comment (jrgonzalezg) [inline gradle.properties:41]: Add missing new line

File: gradle.properties [hunk 4]
```diff
@@ -33,6 +33,8 @@ POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=gcacace
 POM_DEVELOPER_NAME=Gianluca Cacace
 
-ANDROID_BUILD_TARGET_SDK_VERSION=25
-ANDROID_BUILD_TOOLS_VERSION=25.0.3
```
Comment (jrgonzalezg) [inline gradle.properties:37]: Why remove this? Newest one should be 29.0.3

File: SignaturePad-Example/build.gradle [hunk 5]
```diff
@@ -21,6 +20,6 @@ android {
 }
 
 dependencies {
-    compile project(":signature-pad")
-    compile 'com.android.support:appcompat-v7:25.3.1'
+    implementation project(":signature-pad")
+    implementation 'androidx.appcompat:appcompat:1.0.0'
```
Comment (jrgonzalezg) [inline SignaturePad-Example/build.gradle]: Should be 1.1.0 to avoid issues using it on projects with newer versions.

**Review by jrgonzalezg — CHANGES_REQUESTED** [review]
Just some things on the dependency versions.

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (voghDev) [reply gradle/wrapper/gradle-wrapper.properties]: I'll try :)

File: gradle.properties (hunk 4 above)
Comment (voghDev) [reply gradle.properties:37]: Yes, I removed it as gradle seems to use the proper build tools version without specifying, but if it's better, we can specify 29.0.3

https://stackoverflow.com/questions/46999419/android-studio-3-0-buildtoolsversion-not-found-in-gradle-files

https://developer.android.com/studio/releases/gradle-plugin#3-0-0

Comment (jrgonzalezg) [reply gradle.properties:37]: Not sure, they mention the minimum required, but don't say if that means the latest one with latest bug fixes. As you prefer.

**Review by jrgonzalezg — APPROVED** [review]

File: gradle.properties (hunk 4 above)
Comment (voghDev) [reply gradle.properties:37]: I guess both build tools have bugs 😅 so let's go with the latest ones. Switching to 29.0.3

**Pull request merged**